modules/exploits/windows/http/zoho_password_manager_pro_xml_rpc_rce.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::JavaDeserialization
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Apache OFBiz XML-RPC Java Deserialization',
+        'Description' => %q{
+          This module exploits a Java deserialization vulnerability in Zoho ManageEngine Pro
+          before 12101 and PAM360 before 5510. Unauthenticated attackers can send a
+          crafted XML-RPC request containing malicious serialized data to /xmlrpc to
+          gain RCE as the SYSTEM user.
+        },
+        'Author' => [
+          'Vinicius', # Discovery
+          'Y4er', # Writeup
+          'Grant Willcox' # Exploit
+        ],
+        'References' => [
+          ['CVE', '2022-35405'],
+          ['URL', 'https://xz.aliyun.com/t/11578'], # Writeup
+          ['URL', 'https://www.manageengine.com/products/passwordmanagerpro/advisory/cve-2022-35405.html'], # Advisory
+          ['URL', 'https://archives2.manageengine.com/passwordmanagerpro/12101/ManageEngine_PasswordManager_Pro_12100_to_12101.ppm'] # The patch.
+        ],
+        'DisclosureDate' => '2022-06-24', # Vendor release date of patch and new installer, as advisory lacks any date.
+        'License' => MSF_LICENSE,
+        'Platform' => ['win'],
+        'Arch' => [ARCH_CMD, ARCH_X64],
+        'Privileged' => true,
```

Comment:
```suggestion
        'Privileged' => false,
```

Reply:
Why? This gets SYSTEM privileges.

Reply:
Apologies, my mistake, I was confused as to what this denoted.

Reply:
All good I was also similarly confused on this is the past 👍